net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -394,13 +409,14 @@ mwan3_set_general_iptables()
 					-p ipv6-icmp \
 					-m icmp6 --icmpv6-type 137 \
 					-j RETURN
-				# do not mangle outgoing echo request
-				$IPT6 -A mwan3_hook \
-					-m set --match-set mwan3_source_v6 src \
-					-p ipv6-icmp \
-					-m icmp6 --icmpv6-type 128 \
-					-j RETURN
-
+				if [ "$FIX_KERNEL_ACTUAL_SK_BUG" = true ]; then
```

Comment:
```suggestion
				if "$FIX_KERNEL_ACTUAL_SK_BUG"; then
```

Reply:
What don't you like about it? [Shellcheck](https://www.shellcheck.net/) does not throw a warning. I have also read on the Internet that this is better. 
I think that is a matter of taste.

Reply:
IMO, it doesn't read naturally.  _if <something>_ reads much more naturally than _if something is [not] something else_.

i.e. _if cold_ reads much more naturally than _if temperature is lower than $warm_.  It's more similar to how we speak as humans and less like how machines want to speak.

Reply:
I usually do if just like suggested. However, the fix_kernel variable as an action and not a property sounds strange.

if $KERNEL_HAS_BUG_xxxx, $KERNEL_NEEDS _FIX_FOR_.... or something similar would be better

Reply:
got it thanks